invenio_communities/assets/semantic-ui/js/invenio_communities/community/frontpage.js

```diff
@@ -181,12 +25,14 @@ if (userCommunitiesContainer) {
     userCommunitiesContainer
   );
 }
-ReactDOM.render(
-  <CommunitiesCardGroup
-    fetchDataUrl="/api/communities?q=&sort=newest&page=1&size=5"
-    emptyMessage="There are no featured communities."
-    defaultLogo="/static/images/square-placeholder.png"
-  />,
-  featuredCommunitiesContainer
-);
-export default CommunitiesCardGroup;
+
+if (newCommunitiesContainer) {
+  ReactDOM.render(
+    <CommunitiesCardGroup
+      fetchDataUrl="/api/communities?q=&sort=newest&page=1&size=5"
+      emptyMessage="There are no new communities."
```

Comment:
I missed another thing... sorry :(. Shall we make this translatable? WDYT?
We can merge like this and I can push a commit later on.

Reply:
Yeah I just followed what was there and I am not too familiar with the translation on the frontend. I made a separate ticket for it: https://github.com/inveniosoftware/invenio-communities/issues/1073 (could even be expanded to a "translation pass" before next release). Will merge as-is for now :+1: .